.github/workflows/dependabot-alerts.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Dependabot Alerts Action'
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      # X.X.X - Latest version available at: https://github.com/kunalnagarco/action-cve/releases
+      - uses: kunalnagarco/action-cve@v1.6.14
+        with:
+          token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

Comment:
I created a PAT from my user account `Andrew5194` but not sure if this should be a PAT from another account within the FAS organization.